examples/README.md

```diff
@@ -0,0 +1,23 @@
+The section provides examples of solving various equations using QuDynamics.
+
+Currently we have the following two examples :
+
+1. Quantum Harmonic Oscillator
+2. Jaynes Cummings Model
+
+We would like to thank QuTiP developers for sharing the Jaynes Cummings Model, the
+current implementation is a direct porting of QuTiP JC model to QuDynamics version.
+
+Here are some QuDynamics versions (with results) and their QuTiP references :
+
+1. Implementation of Jaynes Cummings Model
+
+[QuDynamics version](https://github.com/amitjamadagni/Notes-Notebooks/blob/master/Notebooks/JC%20model%20-%20%20QuTiP%20in%20QuDynamics.ipynb)
```

Comment:
You forgot to change the first two links. Could you also change the file name "JC_model ..." to "JC_Model ..." Or vice versa?